content/blog/getting-started-with-the-hpe-data-services-cloud-console-powershell-sdk.md

```diff
@@ -0,0 +1,186 @@
+    Write-Host ("Response headers: {0}" -f ($_.Exception.Response.Headers | ConvertTo-Json))    
+}
+```
+
+A﻿fter getting the systems list with the *Invoke-SystemsList* command, I stored the system name and system Id in an array, because I will need the system Id for many of the possible API commands. Generally it is the case that  the systems, volume, hosts, initiators etc. you want to manipulate in your scripts are defined by their ids. I.e. if you work with the DSCC API you will work with the id of the object you want to change or delete. 
+
+If I want to create for instance a new volume, then I do need to define the volume parameters like name, size etc. and the system where I want to create the volume. The system is defined by its system id and a new volume can be created with the *Invoke-VolumeCreate* command that has two input parameters: the system id and the volume parameters that are stored in a *CreateVolumeInput* object using the *Initialize-CreateVolumeInput* method. 
+
+```powershell
+$SystemId = $System.Primera650
+$CreateVolumeInput = Initialize-CreateVolumeInput -Comments "DSCC API -Thomas Beha" `
+  -Count 1 `
+  -DataReduction $true `
+  -Name "DSCC-API-Vol" `
+  -SizeMib 16384 `
+  -SnapCpg "SSD_r6" `
+  -UserCpg "SSD_r6"  
+try {
+	$Result = Invoke-VolumeCreate -SystemId $SystemId -CreateVolumeInput $CreateVolumeInput
+} catch {
+    Write-Host ("Exception occurred when calling Invoke-CreateVolume: {0}" -f ($_.ErrorDetails | ConvertFrom-Json))
+    Write-Host ("Response headers: {0}" -f ($_.Exception.Response.Headers | ConvertTo-Json))	
+}
+```
+
+T﻿he *Invoke-VolumeCreate* command will return the information of the generated task. You can use the *Get-Task* command to monitor the the progress of this task.
+
+```powershell
+$Status = Get-Task $Result.taskUri
+while( ($Status.progressPercent -lt 100) -and ($Status.state -ne "FAILED")){
+    $Status.progressPercent
+    Start-Sleep -Seconds 5	
+    $Status = Get-Task  $TaskId
+}
+```
+
+These are the steps, I used to create a new Volume on my Primera 650 array with a name of DSCC-API-Vol and a size of 16GB.
+
+![](/img/dscc-api-vol.png "DSCC-API-Vol UI view")
+
+# S﻿ummary
+
+In conclusion, the DSCC PowerShell SDK, derived from the DSCC Open API 3.x spec, includes at the time of writing this blog already 349 commands that can be used to manage your storage with  the DSCC API. It is at the moment the most complete PowerShell tool to manage DSCC managed HPE storage arrays. Since it is also easily updated with new features once they are available - simply recreate it with the most current DSCC Open API 3.x spec - it is the recommended way for customer  to build PowerShell scripts accessing the DSCC API. 
+
+I﻿ hope that the above examples will help in getting you started with the DSCC PowerShell SDK. While I have not tested every possible command, I did not find any issues in the SDK working on the extensive daily tasks list that I received from my customer.
```

Comment:
I would simplify this first sentence to:
"I hope the above examples help get you started with the DSCC PowerShell SDK."

For the second sentence, I would modify it to:
"While I have not tested every possible command, I have not found any issues in working with the SDK on the extensive daily tasks list received from my customer."